README.md

```diff
@@ -28,7 +28,7 @@ Create a new WebSocket connection using the following.
 - Password is optional.  
 
 ```js
-const obs = new OBSWebSocket('address', 'password');
+const obs = new OBSWebSocket({ address: 'localhost:4444', password: '$up3rSecretP@ssw0rd' });
```

Comment:
Sleep apnea is a bitch.
Are you going back in for that study again at some point?